Packs/QualysFIM/Integrations/QualysFIM/QualysFIM_test.py

```diff
@@ -38,18 +58,22 @@ def test_list_events_command(requests_mock) -> None:
      - Ensure outputs prefix is correct.
      - Ensure outputs key fields is correct.
     """
-    from QualysFIM import Client, list_events_command
+    from QualysFIM import list_events_command
+
+    # Set
     mock_response = util_load_json('test_data/list_events.json')
     requests_mock.post(f'{BASE_URL}fim/v2/events/search', json=mock_response)
-    requests_mock.post(f'{BASE_URL}/auth', json={})
-    client = Client(base_url=BASE_URL, verify=False, proxy=False, auth=('a', 'b'))
-    result = list_events_command(client, {'sort': 'most_recent'})
+
+    # Arrange
+    result = list_events_command(qualysfim_client, {'sort': 'most_recent'})
+
+    # Assert
     assert result.outputs_prefix == 'QualysFIM.Event'
-    assert len(result.raw_response) == 2
     assert result.outputs_key_field == 'id'
+    assert result.raw_response == mock_response
```

Comment:
let's test the output too

Reply:
Added `assert` in commit 979e11d2833c78af71e0f9503f65bb2df5220da3